__test__/commentsCounter.test.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * @jest-environment jsdom
+ */
+import countMessages from '../modules/commentsCounter.js';
+
+describe('Count Comments', () => {
+  test('Count four Comments', () => {
+    let comments = '';
+    // eslint-disable-next-line no-plusplus
+    for (let i = 0; i < 4; i++) {
+      comments += '<div class= "comentsByPokemon"></div>';
+    }
+    document.body.innerHTML = comments;
+    const result = countMessages();
+    expect(result).toEqual(4);
+  });
+  test('Count six Comments', () => {
+    let comments = '';
+    // eslint-disable-next-line no-plusplus
```

Comment:
As requested by the previous reviewer, kindly avoid disabling linter checks as much as possible. They represent coding standards that you share with your team and most of the time enforce best coding practices.

_There are about 10 other similar occurrences, kindly remove them too and fix the linter errors that result._